.github/workflows/icfy-stats.yml

```diff
@@ -0,0 +1,53 @@
+name: Calculate ICFY stats
+
+on: [push]
+
+jobs:
+  build:
+    name: Build ICFY stats
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '>=12.15.0'
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Fetch git history
+      run: git fetch --prune --unshallow
+    - name: Install dependencies
+      env:
+        PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
+      run: npm ci
+    - name: Build ICFY stats
+      env:
+        NODE_ENV: production
+        BROWSERSLIST_ENV: defaults
+        WORKERS: 2
+      run: npm run analyze-icfy
+    - run: mkdir icfy-stats && mv client/{chart,stats}.json icfy-stats
+    - uses: actions/upload-artifact@v1
+      with:
+        name: icfy
+        path: icfy-stats
+    - name: Upload build artifact
+      env:
+        ICFY_SECRET: ${{ secrets.ICFY_SECRET }}
+      run: |
+        ANCESTOR_SHA1=$(git merge-base HEAD origin/master)
+        CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+        curl -X POST --globoff                                                     \
+          "http://api.iscalypsofastyet.com:5000/submit-stats?secret=$ICFY_SECRET"  \
```

Comment:
huh. it hated the `${{}}` syntax huh?

Reply:
hmmm, maybe the secret has/had bad URL characters in it?

Reply:
We could inline the secret again without the env, the important bit is the `--globoff` option.

Reply:
Yes, the secret contains characters like `%`, `|`, `}` or `*`. Definitely not URL and shell friendly 🙂